index.html

```diff
@@ -15,43 +15,60 @@
-
-  which there is no center because every point, or none, is center — an equal earth which all men occupy as equals. The airman's earth, if free men make it.
-
-  Let's Go For A Walk
-
-  The Earth was small, light blue, and so touchingly alone, our home that must be defended like a holy relic. The Earth was absolutely round.
-
-  It suddenly struck me that that tiny pea, pretty and blue, was the Earth. I put up my thumb and shut one eye, and my thumb blotted out the planet Earth.
-
-  Sign Up
-
-  <img src="img/space-walk.jpg" alt="Space walking astronaut">
-
-  Copyright SpaceWalkers 2018
+  <div class="header-container">
+    <header class="main-header">
+      <h1>SpaceWalkers</h1>
+      <nav>
+        <a href="#">Home</a>
+        <a href="#">About Us</a>
+        <a href="#">Blog</a>
+        <a href="#">Contact</a>
+        <a href="#">Sign In</a>
+      </nav>
+    </header>
+  </div>
+
+  <div class="container home">
+    <section class="homepage-intro">
+        <img class="homepage-header-img" src="img/main-header.jpg" alt="Space view of Earth">
+    </section>
+    
+    <section class="content-section">
+      <div class="text-content">
+        <h2>Lift Off</h2>
+        <p>As we got further and further away, it [the Earth] diminished in size. Finally it shrank to the size of a marble, the most beautiful you can imagine. That beautiful, warm, living object looked so fragile, so delicate, that if you touched it with a finger it would fall apart. Seeing this has to change a man.</p>
+      </div>
+    </section>
+
+    <section class="content-section middle-launch-images">
+        <img class="blast-img" src="img/blast.jpg" alt="Rocket blasting off">
+        <img class="sky-img" src="img/sky.jpg" alt="Space sky image">
+        <img class="launch-img" src="img/launch.jpg" alt="Launching rocket">
+    </section>
+
+    <section class="content-section">
+      <div class="text-content planets">
+        <h2>Can We Visit Planets?</h2>
+        <p>Never in all their history have men been able truly to conceive of the world as one: a single sphere, a globe, having the qualities of a globe, a round earth in which all the directions eventually meet.</p>
+        <p>which there is no center because every point, or none, is center — an equal earth which all men occupy as equals. The airman's earth, if free men make it.</p>
+      </div>
+    </section>
+
+    <section class="bottom-content">
+      <div class="text-content">
+        <h2>Let's Go For A Walk</h2>
+        <p>The Earth was small, light blue, and so touchingly alone, our home that must be defended like a holy relic. The Earth was absolutely round.</p>
+        <p>It suddenly struck me that that tiny pea, pretty and blue, was the Earth. I put up my thumb and shut one eye, and my thumb blotted out the planet Earth.</p>
+        <div class="sign-up-button">Sign Up</div>
```

Comment:
It doesn't make sense to adjust the size of the `Sign Up` button before shrinking to 500px width